src/script.js

```diff
@@ -1,18 +1,32 @@
-let myTimer;
+const long = 2;
+const medium = 1.5;
+const short = 1;
+const myTimer = 1000;
 
-var start = function () {
-  myTimer = setInterval(updater, 5000);
+let running = false;
+
+var start = async function () {
+  console.log('start');
+  running = true;
+  for (let i = 0; i <= 40 && running; i++) {
```

Comment:
40 is a magic number

Reply:
and this will run for 41 iterations which probably isn't what you intended

Reply:
not a magic number. There are 10 sets of combinations with 4 (a,b,c,d) in each making it 40 combinations in total.
As a future feature I will add belt modifier which will limit the combinations to only those available for that belt and set a new number for this value

Reply:
I replaced it with a variable for easier readability.